packages/webcomponents/src/components/checkout/card-form/card-form.tsx

```diff
@@ -0,0 +1,77 @@
+import { Component, h, Method } from "@stencil/core";
+import { StyledHost } from "../../../ui-components";
+
+@Component({
+  tag: "card-form",
+  shadow: true,
```

Comment:
Does this need shadow since it will always be used in a parent component? Like `Checkout` or `PaymentForm` (soon to be `TokenizePaymentMethod`)

Reply:
Good one! Yeah it doesn't need